packages/x-data-grid-premium/src/hooks/features/aiAssistant/api.ts

```diff
@@ -9,15 +9,15 @@ type Result<T> = { ok: false; message: string } | { ok: true; data: T };
  * @param {string} context The prompt context containing necessary information about the current columns definition.
  * Either use the `context` parameter of the `onPrompt` callback or the return value of the `unstable_getPromptContext()` API method.
  * @param {string} conversationId The id of the conversation the prompt is part of. If not passed, prompt response will return a new conversation id that can be used to continue the newly started conversation.
- * @param {string} additionalContext Optional, additional context to make the processing results more accurate.
+ * @param {PromptResolverOptions} options Optional, includes settings to extend and customize the prompt resolver's behaviour.
  * @returns {Promise<PromptResponse>} The grid state updates to be applied.
  */
 export function gridDefaultPromptResolver(
   url: string,
   query: string,
   context: string,
   conversationId?: string,
-  additionalContext = '',
+  options?: PromptResolverOptions,
```

Comment:
If we want to avoid the breaking change?
```suggestion
  optionsOrAdditionalContext?: string | PromptResolverOptions,
```

Reply:
I like this suggestion 👍🏻

Reply:
All right, I'll align https://github.com/mui/mui-private/pull/986 with this suggestion and we can merge this without a breaking change in the next version, postpone the breaking change to v9

Thanks @MBilalShafi and @cherniavskii for quick clarity*